src/graphql/resolvers/businessInfoResolver.ts

```diff
@@ -1,13 +1,24 @@
+import { BusinessImageProviderAggregator } from '../../providers/data_providers/businessImageProviderAggregator';
+import {
+	ExtractionConfig,
+	getExtractionProvider,
+} from '../../providers/extraction_providers/extractionProviderFactory';
+import { getInterpretationProvider, InterpretationConfig } from '../../providers/interpretation_providers';
 import { BusinessInferences, BusinessInfoInput } from '../generatedTypes';
 
 export async function getBusinessInfo(businessInfo: BusinessInfoInput): Promise<BusinessInferences> {
-	return {
-		servesAlcohol: {
-			insight: false,
-			confidence: 0.5,
-			evidence: {
-				images: [],
-			},
-		},
-	};
+	if (!process.env.EXTRACTION_CONFIG) throw new Error('Undefined environment variable EXTRACTION_CONFIG');
```

Comment:
eventually we should probably do the same for the BusinessImageProviderAggegator; let the caller pass in the config object rather than assuming that it comes from process.env